config/webpack/environment.js

```diff
@@ -1,3 +1,23 @@
-const { environment } = require('@rails/webpacker')
+const { environment } = require('@rails/webpacker');
 
-module.exports = environment
+const customConfig = {
```

Comment:
What does all this configuration mean? Why do you need to include it?

Reply:
Solved it ! Rails `webpacker` works fine with `webpack 4.x.x` so I installed 
```
yarn add webpack@4.44.1

```
and `package.json` now is updated, additionally, the file `config/webpack/environment.js` is set by default.